Updated to use facebook/webdriver 1.0

DavertMik opened the pull request.

javigomez commented:
closes #2120

DavertMik commented:
they have changed naming of every class. This may look like a BC break :(

javigomez commented:
ouch!

Here is the compare view: 

https://github.com/facebook/php-webdriver/compare/v0.5...1.0.0

DavertMik commented:
@zbateson could you look into it? I implemented a bit more than just updating webdriver. Actually I added few new methods to WebDriver and PhpBrowser to make them simpler to use in helpers

DavertMik commented on composer.json:
```diff
@@ -19,7 +19,7 @@
         "phpunit/phpunit": "~4.7.0",
-        "facebook/webdriver": "~0.4|~0.5",
+        "facebook/webdriver": "~0.4|~0.5|~1.0",
         "guzzlehttp/guzzle": ">=4.0|<7.0",
```
oh, probably this should be changed to `~1.0` only

DavertMik commented on composer.json:
```diff
@@ -19,7 +19,7 @@
         "ext-mbstring": "*",
         "phpunit/phpunit": "~4.7.0",
-        "facebook/webdriver": "~0.4|~0.5",
+        "facebook/webdriver": "~0.4|~0.5|~1.0",
```
oh, probably this should be changed to `~1.0` only
